client/v2/common/models/application_logs_response.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+// ApplicationLogsResponse
+type ApplicationLogsResponse struct {
+	// ApplicationId (appidx) application index.
+	ApplicationId uint64 `json:"application-id"`
+
+	// CurrentRound round at which the results were computed.
+	CurrentRound uint64 `json:"current-round"`
+
+	// LogData
+	LogData []ApplicationLogData `json:"log-data,omitempty"`
+
+	// NextToken used for pagination, when making another request provide this token
+	// with the next parameter.
+	NextToken string `json:"next-token,omitempty"`
```

Comment:
how is the nextToken calculated?

Reply:
It's supposed to treated as an opaque string that's only used to pass into the `LookupApplicationLogsByID.Next` method.

In practice I believe it's the TxID of the next transaction response, calculated by postgres: https://github.com/algorand/indexer/blob/9040a2e6c7b79ebda11f1531afd72020963c7849/api/handlers.go#L745